### DISCUSSION TITLE
Spilt Amount Using Google Pay

### DISCUSSION THREAD

**Discussion by dineshMadvise** [flutter-stripe/flutter_stripe#877] [Q&A]
when i use pay plugin and place order it is generate token in stripe platform account (using publish key).

how to create google pay token in Stripe connected account ?

Let me explain my scenario over here

here is the restaurant(Connected Account) and customer

Customer added item for an value of 10$
he clicked on payment method as a google pay and paid 10$

So here out of 10$ we have to split that amount

so out of 10$ it should split amount to our app (xyz) and stripe commission (xyz) as per our fourmula

Example :
out of 10$ it should cut 1$ and transfer to stripe account and 1$ transfer to our app as a commission and rest of the amount should be transferred to restaurants(Connected Account)

Note:
Apple Pay split payment working fine. bcz. in flutter_stripe package provide one method "createApplePayToken". using this method i create token in connected account. (Here i also use pay package)

How to split amount using google pay ?.

please give me solution.

**Comment by remonh87** [comment]
I personally think you should do this in the backend using for example: https://stripe.com/docs/connect/charges-transfers. I think the customer should pay once and then you handle the different charges in the backend.  Let me know if this is what you mean.
